app/models/liquid_partial_observer.rb

```diff
@@ -0,0 +1,10 @@
+class LiquidPartialObserver < ActiveRecord::Observer
+  def after_save(record)
+    LiquidRenderer::Cache.invalidate
```

Comment:
You want me to test what has already been tested: https://github.com/rails/rails-observers/blob/master/test/observing_test.rb#L126-L129